graphjet-core/src/test/java/com/twitter/graphjet/algorithms/counting/TopSecondDegreeByCountTest.java

```diff
@@ -159,7 +164,7 @@ public void testTopSecondDegreeByCountWithSocialProofTypeUnionsWithSmallGraph()
       new RequestedSetFilter(new NullStatsReceiver())
     ));
 
-    TopSecondDegreeByCountRequest topSecondDegreeByCountRequest = new TopSecondDegreeByCountRequest(
+    TopSecondDegreeByCountRequestForTweet topSecondDegreeByCountRequestForTweet = new TopSecondDegreeByCountRequestForTweet(
```

Comment:
Twitter enforces 100 character limit per line for java, and this line looks to me exceeding the limit? you might need to rename the variable.